src/main/device/service.ts

```diff
@@ -1,45 +1,63 @@
 import { aes } from '@internxt/lib';
 import { dialog } from 'electron';
 import fetch from 'electron-fetch';
+
 import os from 'os';
 import path from 'path';
-
+import { addProcessIssue } from '../background-processes/process-issues';
 import { getHeaders } from '../auth/service';
 import configStore from '../config';
 
 export type Device = { name: string; id: number; bucket: string };
 
-export async function getOrCreateDevice() {
-  async function createDevice(deviceName: string) {
-    return fetch(`${process.env.API_URL}/api/backup/deviceAsFolder`, {
-      method: 'POST',
-      headers: getHeaders(true),
-      body: JSON.stringify({ deviceName }),
-    });
-  }
-
-  async function tryToCreateDeviceWithDifferentNames(): Promise<Device> {
-    let res = await createDevice(os.hostname());
+const addUnknownDeviceIssue = (error: Error) => {
+  addProcessIssue({
+    errorName: 'UNKNOWN_DEVICE_NAME',
+    kind: 'LOCAL',
+    name: error.name,
+    action: 'GET_DEVICE_NAME_ERROR',
+    errorDetails: {
+      action: '',
+      message: error.message,
```

Comment:
Will this issue provide enough context about what happened?

Reply:
Yes, the http error is proxied to this function, so if the problem is due to server problems such 500, we will see the error in the logs. Since this is a plain error, some properties are empty because we simply need to match the TS type to match the "issue" type